spec/models/chargeback_container_project_spec.rb

```diff
@@ -0,0 +1,232 @@
+describe ChargebackContainerProject do
+  before do
+    MiqRegion.seed
+    ChargebackRate.seed
+
+    EvmSpecHelper.create_guid_miq_server_zone
+    @ems = FactoryGirl.create(:ems_openshift)
+
+    @project = FactoryGirl.create(:container_project, :name => "my project", :ext_management_system => @ems)
+    @live_group = FactoryGirl.create(:container_group, :name => "live group", :ext_management_system => @ems)
+    @disconnected_group = FactoryGirl.create(:container_group,
+                                             :name                  => "disconnected group",
+                                             :ext_management_system => @ems)
+    @project.container_groups << @live_group
+    @project.container_groups << @disconnected_group
+
+    @cbr = FactoryGirl.create(:chargeback_rate, :rate_type => "compute")
+    temp = {:cb_rate => @cbr, :object => @ems}
+    ChargebackRate.set_assignments(:compute, [temp])
+
+    @hourly_rate       = 0.01
+    @count_hourly_rate = 1.00
+    @cpu_usage_rate    = 50.0
+    @cpu_count         = 1.0
+    @memory_available  = 1000.0
+    @memory_used       = 100.0
+    @net_usage_rate    = 25.0
+
+    @options = {:interval_size       => 1,
+                :end_interval_offset => 0,
+                :tag                 => "/managed/environment/prod",
+                :ext_options         => {:tz => "Pacific Time (US & Canada)"},
+                :entity_id           => @project.id
+    }
+
+    @disconnected_group.disconnect_inv
```

Comment:
@simon3z This covers a disconnected group scenario. But do you think theres something im not testing here? (Ill add it in a subsequent PR)